Assets/Game/Scripts/Cube.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Cube : MonoBehaviour
```

Comment:
It would be great if we could rename it to other meaningful class name.

Reply:
Agree..   but let rename later...   because I think the art team will move the block...   And we can rename the class along with the block.

Reply:
Sounds good!